src/flow/darcy_flow_mh.cc

```diff
@@ -712,17 +719,14 @@ void DarcyMH::allocate_mh_matrix()
         ls->mat_set_values(loc_size, local_dofs, loc_size, local_dofs, zeros);
         
 
-        std::vector<int> tmp_rows;
-        tmp_rows.reserve(200);
-
         // compatible neighborings rows
         unsigned int n_neighs = ele_ac.full_iter()->n_neighs_vb;
-        for (unsigned int i = 0; i < n_neighs; i++) {
+        for (i = 0; i < n_neighs; i++) {
             // every compatible connection adds a 2x2 matrix involving
             // current element pressure  and a connected edge pressure
             Neighbour *ngh = ele_ac.full_iter()->neigh_vb[i];
             int neigh_edge_row = mh_dh.row_4_edge[ ngh->edge_idx() ];
-            tmp_rows.push_back(neigh_edge_row);
+            tmp_rows[i] = neigh_edge_row;
```

Comment:
This is wrong, reserve is not resize. tmp_rows have still size 0.

Reply:
agree, my bad..